src/datastore/postgres-store.ts

```diff
@@ -820,9 +820,29 @@ export class PgDataStore
       UPDATE names
       SET canonical = $2
       WHERE index_block_hash = $1 AND canonical != $2
+      RETURNING name
       `,
       [indexBlockHash, canonical]
     );
+    nameResult.rows.forEach(async name => {
+      await client.query(
+        `
+        WITH latest_name AS (
+          SELECT id
+          FROM names
+          WHERE name = $1 AND canonical = true
+          ORDER BY registered_at DESC LIMIT 1
+        )
+        UPDATE names 
+        SET latest = false
+        WHERE name = $1 AND canonical =true
+        
+        UPDATE names 
+        SET latest = true 
+        WHERE id = latest_name.id`,
```

Comment:
This looks like sql code accidentally left in?

Reply:
No! The previous query set all latest to false, this one set only the latest to be true.